frontend/src/views/Bridge.vue

```diff
@@ -6,7 +6,7 @@
     <div class="row m-3 justify-content-center">
       <h3>{{$t('bridge.transferNftsToAnotherChain')}}</h3>
     </div>
-    <div class="row mt-3 justify-content-center">
+    <div class="bridge-transfer-fee row mt-3 justify-content-center">
```

Comment:
(Optional) This would be a bit better here imo:
```suggestion
    <div class="w-100 row mt-3 justify-content-center">
```